src/components/Insertion/index.tsx

```diff
@@ -0,0 +1,216 @@
+import { useState } from 'react';
+import { getDatabase, ref, set, push } from 'firebase/database';
+import app from '../../firebaseConfig';
+
+const Insertion: React.FC = () => {
+  const [insertionData, setInsertionData] = useState({
+    instrumentType: '',
+    model: '',
+    description: '',
+    rentalPrice: '',
+    condition: '',
+    location: '',
+    deliveryMethod: '',
+    notes: '',
+    images: null as FileList | null,
+  });
+
+  const handleChange = (
+    e: React.ChangeEvent<
+      HTMLInputElement | HTMLTextAreaElement | HTMLSelectElement
+    >,
+  ) => {
+    const { name, value } = e.target;
+    setInsertionData({ ...insertionData, [name]: value });
+  };
+
+  const handleFileChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (e.target.files) {
+      setInsertionData({ ...insertionData, images: e.target.files });
+    }
+  };
+
+  const addInsertion = async () => {
+    const db = getDatabase(app);
+    const newInsertionRef = push(ref(db, 'Insertion'));
```

Comment:
di questo c'e da discuterne ma magari dopo che metti il form, tieni conto che io non ti avrei manco fatto fare il push a db in questa PR. ci sta farlo non e' troppo lavoro ma volevo darti un attimo di contesto prima

Reply:
Ok perfetto, allora dopo che sistemo il form approfondiamo